### PR TITLE
feat(release-runbook): added some UX improvements while running through the runbook

### PR DESCRIPTION
**Pull request by kevinawoo**
Just a few more changes while working through the patch release process

### CODE REVIEW

**Review by maggieneterval — APPROVED** [review]
Thanks, @kevinawoo!

File: community/releases/release-manager-runbook/index.md [hunk 1]
```diff
@@ -191,11 +191,24 @@ before merging.
     1. Add a file 1.nn.0.md (e.g., `1.18.0.md`) to hold the release notes for
     the new release.
 
-    1. Copy the changes for this release from the raw build changelog to the new
-    1.nn.0.md file.
+        Use this template to build the file.
```
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
        Use this template to build the file:
```

File: community/releases/release-manager-runbook/index.md [hunk 2]
```diff
@@ -208,8 +221,9 @@ before merging.
     1. **Spinnaker Release Alias** should be the name of a Netflix original TV
     show converted to an alphanumeric string
     (e.g., "Gilmore Girls A Year in the Life").
+    Only worry about name collisions with the current active releases (i.e. `hal version list`)
```
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
    The name must be unique among current active releases (releases returned by `hal version list`).
```

File: community/releases/release-manager-runbook/index.md [hunk 3]
```diff
@@ -208,8 +221,9 @@ before merging.
...
 
-    1. **BOM version** should be "release-1.nn.x-latest-validated" (replacing nn
+    1. **BOM version** should be `release-1.nn.x-latest-unvalidated` (replacing nn
```
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]: s/should/must/
or
s/should/will/
or
s/should/is/

Whichever makes the most sense to you. But saying "should" implies it's optional.

**Review by dorbin — COMMENTED** [review]
A few suggestions, @kevinawoo

File: community/releases/release-manager-runbook/index.md [hunk 4]
```diff
@@ -208,8 +221,9 @@ before merging.
     1. **Spinnaker Release Alias** should be the name of a Netflix original TV
     show converted to an alphanumeric string
     (e.g., "Gilmore Girls A Year in the Life").
+    The name must be unique among current active releases (releases returned by `hal version list`).
 
-    1. **BOM version** should be "release-1.nn.x-latest-validated" (replacing nn
+    1. **BOM version** should be `release-1.nn.x-latest-unvalidated` (replacing nn
```
Comment (kevinawoo) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
    1. **BOM version** is be `release-1.nn.x-latest-unvalidated` (replacing nn
```

File: community/releases/release-manager-runbook/index.md [hunk 5]
```diff
@@ -208,8 +221,9 @@ before merging.
     1. **Spinnaker Release Alias** should be the name of a Netflix original TV
     show converted to an alphanumeric string
     (e.g., "Gilmore Girls A Year in the Life").
+    The name must be unique among current active releases (releases returned by `hal version list`).
 
-    1. **BOM version** should be "release-1.nn.x-latest-validated" (replacing nn
+    1. **BOM version** is be `release-1.nn.x-latest-unvalidated` (replacing nn
```
Comment (kevinawoo) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
    1. **BOM version** is be `release-1.nn.x-latest-unvalidated` (replace nn
```

**Comment by kevinawoo** [comment]
👋 Hi hi @dorbin, comments addressed!

File: community/releases/release-manager-runbook/index.md [hunk 6]
```diff
@@ -208,8 +221,9 @@ before merging.
     1. **Spinnaker Release Alias** should be the name of a Netflix original TV
     show converted to an alphanumeric string
     (e.g., "Gilmore Girls A Year in the Life").
+    The name must be unique among current active releases (releases returned by `hal version list`).
 
-    1. **BOM version** should be "release-1.nn.x-latest-validated" (replacing nn
+    1. **BOM version** is be `release-1.nn.x-latest-unvalidated` (replace nn
```
Comment (dorbin) [inline community/releases/release-manager-runbook/index.md]:
```suggestion
    1. **BOM version** is `release-1.nn.x-latest-unvalidated` (replace nn
```

Comment (dorbin) [reply community/releases/release-manager-runbook/index.md]: I _should_ have said `s/should be/is/

Oops :)

**Review by dorbin — APPROVED** [review]
One correction to my own suggestion. But LGTM, and thanks @kevinawoo !

**Pull request merged**